set-recording-comments.user.js

```diff
@@ -11,6 +11,7 @@
 // @match          *://*.mbsandbox.org/release/*
 // @exclude        *musicbrainz.org/release/*/*
 // @exclude        *.mbsandbox.org/release/*/*
```

Comment:
mbsandbox.org doesn't seem to be online anymore: could you remove the two mentions here and on L.11?

Reply:
Sure thing, I have now removed them and made the match/exclude patterns consistent. I was not sure whether these sandboxes were internal servers or so and hesitated to remove something that did no harm.